backend/audit/models.py

```diff
@@ -294,11 +310,12 @@ def transition_to_submitted(self):
         the appropriate privileges will done at the view level.
         """
 
-        from audit.etl import ETL
+        # from audit.etl import ETL
```

Comment:
**question**: Is there an issue here with multiple workstreams that are not merged in yet? Why would we be commenting out a whole piece of functionality in this PR that is otherwise adding things?

Reply:
Good catch - this was commented out earlier on when that ETL stuff was (I think) in a broken state